src/coreclr/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/ExecutionEnvironmentImplementation.MappingTables.cs

```diff
@@ -719,7 +719,7 @@ private unsafe bool TryGetMethodForOriginalLdFtnResult_InvokeMap_Inner(NativeFor
                 QTypeDefinition qTypeDefinition = GetMetadataForNamedType(declaringTypeHandleDefinition);
 
                 MethodHandle nativeFormatMethodHandle =
-                    (((int)HandleType.Method << 24) | (int)entryMethodHandleOrNameAndSigRaw).AsMethodHandle();
+                    (((int)HandleType.Method << 25) | (int)entryMethodHandleOrNameAndSigRaw).AsMethodHandle();
```

Comment:
Should we have `25` defined as a constant somewhere so that it is not hardcoded in so many places?

Reply:
I can do it if you insist but it doesn't seem like a great use of time (especially given we'd need to also do the same for 25/7/1FFFFFF/7F and we'd need to include this from like 5 csproj files). It's unlikely we're going to change this ever again. The existing numbers survived for longer than my tenure on the .NET team.

Reply:
I do not have a strong opinion